02.extensions/zoomailing/02.troubleshooting/02.zoolingual-integration/docs.md

```diff
@@ -0,0 +1,13 @@
+---
+title: ZOOlingual Integration
+taxonomy:
+    category: docs
+---
+
+On multilanguage sites, which have both ZOOmailing and ZOOlingual, preview and sent newsletters are displayed in the Joomla admin language.
```

Comment:
When we preview the newsletter that uses ZOOmailing tag, the newsletter will be displayed in the current language admin logg in